packages/extension/src/dashboard-extension.spec.ts

```diff
@@ -17,42 +17,101 @@
  ***********************************************************************/
 
 import type { WebviewPanel, ExtensionContext } from '@podman-desktop/api';
-import { Uri, window } from '@podman-desktop/api';
-import { beforeEach, test, vi } from 'vitest';
+import { kubernetes, Uri, window } from '@podman-desktop/api';
+import { assert, beforeEach, describe, expect, test, vi } from 'vitest';
 import { DashboardExtension } from './dashboard-extension';
+import { vol } from 'memfs';
 
-import { readFile } from 'node:fs/promises';
+import type { ContextsManager } from './manager/contexts-manager';
+import type { ContextsStatesDispatcher } from './manager/contexts-states-dispatcher';
 
 let extensionContextMock: ExtensionContext;
 let dashboardExtension: DashboardExtension;
+let contextsManagerMock: ContextsManager;
+let contextsStatesDispatcher: ContextsStatesDispatcher;
 
-vi.mock(import('node:fs'));
-vi.mock(import('node:fs/promises'));
+vi.mock('node:fs');
```

Comment:
we're using vi.mock(import(')) pattern to ensure it's working well if we move/change a dependency